src/test/resources/liquibase/harness/change/expectedSnapshot/cloudspanner/createTableWithGeneratedColumn.json

```diff
@@ -0,0 +1,16 @@
+{
+  "snapshot": {
+    "objects": {
+      "liquibase.structure.core.Column": [
+        {
+          "column": {
+            "name": "FullName",
+            "type": {
+              "typeName": "STRING(400)"
```

Comment:
The snapshot does not contain any information about the fact that this is a generated column. I'm not sure if Liquibase even supports that, so let's leave it as-is in this PR. Created #416 to track this.